content/en/flux/security/secrets-management.md

```diff
@@ -144,7 +144,7 @@ cluster is also an option. Which could be a useful alternative when lifting and
 shifting legacy applications that already depend on such approach.
 
 Here the secret management solution will become a single point of failure,
-expect issues when it goes temporarily unavailable and have disaster recovery plans.
+expect issues when it goes temporarily unavailable and has disaster recovery plans.
```

Comment:
```suggestion
expect issues when it goes temporarily unavailable and have disaster recovery plans.
```

This is desired. It is a call to action to the user to "have disaster recovery plans" available. Maybe we could make it clearer this way:

```suggestion
expect issues when it goes temporarily unavailable and make sure to have disaster recovery plans.
```